Js files

gwengibbes opened the pull request.

Commit:
recipe generator js file

Created a specific js file for the recipe generator page

Commit:
Update js files

Created independent js files for each page.

Commit:
Updated html files

Included specific JavaScript for each page.

Commit:
Updated CSS

Reformatted style on CSS

Commit:
Merge branch 'wine_glass_logo' into js_files

ElodieLongo commented on js/wine-pairing.js:
```diff
@@ -0,0 +1,22 @@
+    const wines= await fetchWines(wineFamily)
+    console.log(wines)
+
+    // Populae the drop downlist with the list of wines from the API 
```
This is the part I am not sure about. From my analysis of the Spoonacular API, I think it is easyer to have a manually selected array of wines to populate the select (dropdowns) as I did today. See current index.js. Let me know if you have something different in mind and let's have a recap meeting. Thank you!

Commit:
Added image and css styling

 resolves issue #9

Commit:
Added Image to Recipe Generator Page

Resolves issue #10 & #18

Commit:
Resolved issue with page linking urls

Commit:
Merge branch 'main' into js_files

Commit:
Update wine-pairing.js

Ensure that the drop down list is populated on the dropdown page. Moved code to wine-pairing js file.

jordanatuf reviewed (approved):
Looks good to me!
